libtransmission/rpc-server.cc

```diff
@@ -664,7 +664,7 @@ static bool bindUnixSocket(struct event_base* base, struct evhttp* httpd, char c
     tr_logAddNamedError(
         MyName,
         _("Unix sockets are not supported on Windows. Please change \"%s\" in your configuration file."),
-        TR_KEY_rpc_bind_address);
+        tr_quark_get_string(TR_KEY_rpc_bind_address, nullptr));
```

Comment:
One other thing that showed up in the CI Windows builds, the args to this function should be flagged with `[[maybe_unused]]` to silence the four "unused argument" warnings on _WIN32 builds

```
static bool bindUnixSocket(
    [[maybe_unused]] struct event_base* base,
    [[maybe_unused]] struct evhttp* httpd,
    [[maybe_unused]] char const* path,
    [[maybe_unused]] int socket_mode)
```